app/services/authentication/authenticator.rb

```diff
@@ -73,6 +83,16 @@ def call
 
     private
 
+    def guard_against_spam_from!(identity:)
+      domain = identity.email.split("@")[-1]
+      return unless domain
+      return if Settings::Authentication.acceptable_domain?(domain: domain)
+
+      message = "Marking as spam email #{identity.email} from provider #{identity.provider}."
```

Comment:
**(non-blocking, suggestion)**: 

@jeremyf I don't have a strong opinion on this but I found the error message a bit confusing.

I couldn't actually test this on the interface because I don't have OAuth setup and the setup was taking some time but looking at the code, from my understanding this message will be shown to a user when they're trying to signup from domain that has already been blocked by. a Forem admin. 

If this is the case should we update the message to be a bit more explicit and use a past tense in the message as the domain was already marked as spam prior to this signup.

```suggestion
      message = "This #{identity.provider} email address #{identity.email} has been marked as spam"
```